### PR TITLE
Add energy_scale (kWh, MWh) option

### PR DESCRIPTION
**Pull request by alaraun**

### CODE REVIEW

File: custom_components/entsoe/sensor.py [hunk 1]
```diff
@@ -44,13 +46,13 @@ class EntsoeEntityDescription(SensorEntityDescription):
...
 
 
-def sensor_descriptions(currency: str) -> tuple[EntsoeEntityDescription, ...]:
+def sensor_descriptions(currency: str, energy_scale: str) -> tuple[EntsoeEntityDescription, ...]:
     """Construct EntsoeEntityDescription."""
     return (
         EntsoeEntityDescription(
             key="current_price",
             name="Current electricity market price",
-            native_unit_of_measurement=f"{currency}/{UnitOfEnergy.KILO_WATT_HOUR}",
```
Comment (alaraun) [inline custom_components/entsoe/sensor.py:53]: UnitOfEnergy.KILO_WATT_HOUR as it will be deprecated by HA 2025.1

**Comment by Roeland54** [comment]
Reviewed and tested. Thanks for your contribution.

**Comment by alaraun** [comment]
Is it also possible to make a release?
Thanks

**Comment by Roeland54** [comment]
I probably make a new release in the next 2 weeks. 
If you want to use it now just overwrite the files in the custom components folder with the current state of the main branch.

**Comment by Roeland54** [comment]
needed to release a hotfix today. So this also got released.

**Pull request merged**